simcon_project/conversation_templates/models/template_node_choice.py

```diff
@@ -6,6 +6,6 @@ class TemplateNodeChoice (models.Model):
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     choice_text = models.CharField(max_length=500)
     destination_node = models.ForeignKey('conversation_templates.TemplateNode', default=0, related_name='parent_choices', on_delete=models.DO_NOTHING)
-    parent_template = models.ForeignKey('conversation_templates.TemplateNode', default=0, related_name='choices', on_delete=models.DO_NOTHING)
+    parent_template = models.ForeignKey('conversation_templates.TemplateNode', default=0, related_name='choices', on_delete=models.DO_NOTHING) # TODO this needs to be renamed
```

Comment:
This needs to be renamed in models. Everyone loves migrations amiright?

Reply:
why does it need to be renamed?

Reply:
it should be parent_template_node not parent_teplate